pkg/cephfs/volumemounter.go

```diff
@@ -288,6 +290,11 @@ func mountKernel(ctx context.Context, mountPoint string, cr *util.Credentials, v
 	if volOptions.KernelMountOptions != "" {
 		optionsStr += fmt.Sprintf(",%s", volOptions.KernelMountOptions)
 	}
+
+	if !strings.Contains(volOptions.KernelMountOptions, netDev) {
+		optionsStr += fmt.Sprintf(",%s", "_netdev")
```

Comment:
Nit: why not re-use your constant here since you tested against it in the previous line?

Reply:
done